proxy/proxyUtils.js

```diff
@@ -58,7 +59,7 @@ const decorateRequestHeadersForPutApi = function () {
   }
 }
 
-const handleSessionExpiry = (proxyRes, proxyResData, req, res, error, data) => {
+const handleSessionExpiry = (proxyRes, proxyResData, req, res, error, data, triggerNotification = false) => {
```

Comment:
You should not require the flag. You should check the constant object for the request URL has notificationObject, if it is present then trigger notification.